bull-transformer-generator/transformer-generator-core/src/test/java/com/hotels/beans/generator/core/mapping/BeanPropertyTest.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (C) 2019-2020 Expedia, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.beans.generator.core.mapping;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+import java.lang.reflect.Method;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.hotels.beans.generator.core.sample.javabean.Source;
+
+public class BeanPropertyTest {
```

Comment:
missing javadoc

Reply:
Thanks, I thought it was not necessary since CS let me pass.
The supppression.xml file contains this rule:
`<suppress checks="JavadocMethod|JavadocType|JavadocVariable|JavadocStyle|TodoComment"
              files=".*[\\/]src[\\/](test|it)[\\/]|.*Test.java"/>`
Should I add it anyway?

Reply:
yes please, I would be also grateful if you remove the `JavadocStyle` from the suppression